frontend/app/services/TouchpointBackend.scala

```diff
@@ -107,10 +107,8 @@ object TouchpointBackend {
     )
   }
 
-  val Normal = TouchpointBackend(BackendType.Default)
-  val TestUser = TouchpointBackend(BackendType.Testing)
-
-  val All = Seq(Normal, TestUser)
+  lazy val Normal = TouchpointBackend(BackendType.Default)
```

Comment:
well, as soon as you try to use it, it will fall over anyway., I don't see there's a big benefit to blowing up the whole app (although in reality that's what happens anyway due to other parts of the code)
I think only the thing you are actually using should need to work, unless you are needing as a gate/test in which case you should have something more specific than just blowing up the app as a side effect.  The healthcheck does check the normal so we're good to go there.
Although I guess it's more a balance, just which way we'd prefer to err on the side.  But if someone breaks dev it would still be nice to have the ability to test locally in UAT in order to get changes out.